### PR TITLE
Release 2nd Jan 22

### PR DESCRIPTION
**Pull request by abskrj**
1. Generate Multi bitrate video files [#5]
2. Integrated Kafka [#11]
3. Pre Signed URL Endpoint [#4]
4. Initial Test Cases [#21]

### CODE REVIEW

File: controllers/kafka.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+type Video struct {
+	ID   string `json:"id"`
+	Src  string `json:"src"`
+	Type string `json:"type"`
+}
+
+type Body struct {
+	Video Video `json:"video"`
+}
```
Comment (uzaxirr) [inline controllers/kafka.go:20]: We should create a new package `types/types.go` that will hold all the structs like these and the Response types as well

File: controllers/kafka.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+func PublishMessage(c *gin.Context) {
+	var request Body
+	if err := c.ShouldBindJSON(&request); err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		return
+	}
+	jsonBytes, err := json.Marshal(request)
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		return
```
Comment (uzaxirr) [inline controllers/kafka.go:31]: Try to add comments

**Review by uzaxirr — APPROVED** [review]

**Comment by uzaxirr** [comment]
It would also be great if we could have some screen shots of working product in Readme

**Comment by uzaxirr** [comment]
I'll suggest not to make a release until we have a fair amount of test coverage and have addressed all the linting and all the deepsource static analysis issues

**Comment by abskrj** [comment]
Sure, let wait for the 1st release

**Comment by abskrj** [comment]
Closing this for now

**Pull request closed without merging**